frontend/src/app/shared/services/aethel-api.service.ts

```diff
@@ -52,7 +52,7 @@ export class AethelApiService
             }
 
             return this.http
-                .get<AethelList | null>(`${environment.apiUrl}aethel/`, {
+                .get<AethelList | null>(`${environment.apiUrl}aethel`, {
```

Comment:
The user doesn't see this, but devs with the dev console open do (and it hurts my eyes).